backend/copr_backend/sign.py

```diff
@@ -54,7 +52,7 @@ def call_sign_bin(cmd, log):
     return handle.returncode, stdout, stderr
 
 
-def get_pubkey(username, projectname, log, outfile=None):
+def get_pubkey(username, projectname, log, opts, outfile=None):
```

Comment:
Thank you for the PR!

Can you please just accept the `sign_domain` argument, instead of semantically hard-to-define "opts" argument?  And define the docstring?

Reply:
Ok, please take a look.